fullstack/chenxin/shortlink/.env

```diff
@@ -0,0 +1,3 @@
+MYSQL_USERNAME=root
+MYSQL_PASSWORD=1qaz@WSX
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- e475b01e-1ae6-4864-94ec-27ee694f930b -->